packages/devextreme/js/__internal/ui/splitter/utils/layout.ts

```diff
@@ -377,6 +377,10 @@ function adjustAndDistributeLayoutSize(
   return nextLayout;
 }
 
+export function getSafeSize(size: number, fallBackSize: number): number {
+  return typeof size === 'number' ? size : fallBackSize;
```

Comment:
Better to use Number.isFinite(value). I think that this suits for us requirenment